plugins/inputs/wireguard/README.md

```diff
@@ -0,0 +1,42 @@
+# Wireguard Input Plugin
+
+This plugin provides information about Wireguard interfaces
+
+### Configuration
+
+Make sure that running `wg show` on the host works correctly before enabling this plugin
+
+```toml
+
+[[inputs.wireguard]]
+  # if none are provided, all of the available ones will be scraped
+  interfaces = ["wg0"]
+
+```
+
+### Metrics
+
+- wireguard
+  - tags:
+    - name
+    - type
+    - serverpublikey
```

Comment:
Spelling, `serverpublickey`

Reply:
We prefer snake cased tag/field names as well